platform-sdk/swirlds-cli/src/main/java/com/swirlds/cli/utility/ParameterizedClass.java

```diff
@@ -65,8 +64,7 @@ protected Path pathMustExist(final Path path) {
     /**
      * Ensure that a path from the command line does not exist.
      *
-     * @param path
```

Comment:
we really need to do something about this noise generated by formatting. @nathanklick @poulok is there a plan for this?